torch_geometric/transforms/line_graph.py

```diff
@@ -56,6 +62,10 @@ def __call__(self, data: Data) -> Data:
             data.edge_index = torch.stack([row, col], dim=0)
             data.x = data.edge_attr
             data.num_nodes = edge_index.size(1)
+# ============change=====================
+            src, dst = data.edge_index
+            data.edge_attr = line_graph_edge_attr[src]
```

Comment:
Will this work as expected?
lets say the graph originally had E edges and the line graph has E* edges. Then the shape of `line_graph_edge_attr` is (E,-1), and if E* is greater than E then wouldn't this throw an index error.

Reply:
I don't think it will happen, because the data.edge_index here has been converted to the edge_index in the linegraph, so it can run normally.

Reply:
Can we add a test to `test/transforms/test_line_graph.py` to ensure that?